src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3355,7 +3355,7 @@
       </PropertyComment>
     </Field>
   </AbstractNode>
-  <Node Name="ClassDeclarationSyntax" Base="TypeDeclarationSyntax">
+  <Node Name="ClassDeclarationSyntax" Base="TypeDeclarationSyntax" SkipConvenienceFactories="true">
```

Comment:
what's the impact of SkipConvenienceFactories?

Reply:
> what's the impact of SkipConvenienceFactories?

Responded at location that illustrates the impact. See [https://github.com/dotnet/roslyn/pull/65720#discussion_r1038641122](https://github.com/dotnet/roslyn/pull/65720#discussion_r1038641122)